spark/src/main/scala/org/apache/mahout/drivers/RowSimilarityDriver.scala

```diff
@@ -135,7 +133,7 @@ object RowSimilarityDriver extends MahoutSparkDriver {
   }
 
   override def process: Unit = {
-    start()
+    start
```

Comment:
The function is an Arity-0 method. Perhaps I misread: http://docs.scala-lang.org/style/method-invocation.html

Arity-0
Scala allows the omission of parentheses on methods of arity-0 (no arguments):